folding/src/expressions.rs

```diff
@@ -68,7 +66,18 @@ pub enum ExpExtension {
 }
 
 ///Internal expression used for folding, simplified for that purpose
-pub type FoldingExp<C> = Operations<ExtendedFoldingColumn<C>>;
```

Comment:
IIUC this change is not because of necessity..? You could've left it as it was, but decided to change for better future compatibility?